godtools/App/Features/LessonSwipeTutorial/Presentation/LessonSwipeTutorialViewModel.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  LessonSwipeTutorialViewModel.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 4/7/25.
+//  Copyright © 2025 Cru. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class LessonSwipeTutorialViewModel: ObservableObject {
+    
+    private weak var flowDelegate: FlowDelegate?
+    private let getInterfaceStringsUseCase: GetLessonSwipeTutorialInterfaceStringsUseCase
+    private let getCurrentAppLanguageUseCase: GetCurrentAppLanguageUseCase
+    
+    private var cancellables: Set<AnyCancellable> = Set()
+
+    @Published private var appLanguage: AppLanguageDomainModel = LanguageCodeDomainModel.english.rawValue
+    
+    @Published var interfaceStrings: LessonSwipeTutorialInterfaceStringsDomainModel = LessonSwipeTutorialInterfaceStringsDomainModel.emptyStrings()
```

Comment:
Hey @rachaelblue no change needed here, but one thing I've been doing going forward is marking Published properties that are exposed to the View as ```Published private(set) var```.